server/config/client_test.go

```diff
@@ -319,6 +319,25 @@ func TestGetClientConfig(t *testing.T) {
 				"GiphySdkKey": model.ServiceSettingsDefaultGiphySdkKeyTest,
 			},
 		},
+		{
+			"report a problem values",
+			&model.Config{
+				SupportSettings: model.SupportSettings{
+					ReportAProblemType: model.NewPointer("type"),
```

Comment:
Should this be `link` instead of `type`? If I get it right, it tells the kind of value in `ReportAProblemLink`, right? Which in this case is a link.

Reply:
At this level of testing, it doesn't matter what value you add, it is just testing that the value is getting through.

The exception is the link, since (I don't remember where from the top of my head) it gets validated to be a proper link.